modules/preview-react/menu/lib/Menu.tsx

```diff
@@ -78,7 +78,7 @@ const List = styled('ul')({
 });
 
 /**
- * As of Canvas Kit v8, Menu is being deprecated.
+ * @deprecated ⚠️ `DeprecatedMenu` has been deprecated. As of Canvas Kit v8, `Menu` is being deprecated.
```

Comment:
Can we move this `@deprecated` tag to the end of the JSDoc comment and update it for consistency:
```
@deprecated ⚠️ Deprecated Menu has been deprecated and will be removed in a future major version. Please use [Menu](https://workday.github.io/canvas-kit/?path=/docs/components-popups-menu--basic) instead.
```

Also, let's remove the following paragraph since Deprecated Menu is no longer being removed in v10:
```
 * It will be removed in v10. Please see the [upgrade
 * guide](https://workday.github.io/canvas-kit/?path=/story/welcome-upgrade-guides-v8-0--page) for
 * more information.
```

The rest of the JSDoc block can stay as-is.